docs/data-collection/scanning.md

```diff
@@ -257,7 +267,12 @@ It is critical to check that physiological signals are looking good:
 - [ ] While the fieldmap sequence is running,
     - [ ] [Adjust the FoV](scanning-notes.md#setting-the-fov) for the positive-control-task (`func-bold_task-qct_dir-{RL,LR,PA,AP}__cmrr_me4_sms4`) fMRI sequence, and
     - [ ] verify the *Number of measurements* with respect to the [task's timing](intro.md#task-timing) ({{ settings.mri.timings.func_qct }}).
-    - [ ] Verify that the positive-control task `{{ settings.psychopy.tasks.func_qct }}` is open in psychopy, that you calibrated the ET and that the physiological signals are still recording and look ok.
+    - [ ] with a terminal open on the projector's screen, execute the QCT:
```

Comment:
Yesterday's test confirmed that we don't have to open the terminal on the projector screen. We just have to keep the mouse cursor within the projector screen when launching the task.

Reply:
okay, I'm starting a major overhaul of this PR, you are right